src/NUnitTestAdapter/CategoryList.cs

```diff
@@ -82,7 +93,15 @@ public IEnumerable<string> ProcessTestCaseProperties(XmlNode testNode, bool addT
             return categorylist;
         }
 
-        private static bool IsInternalProperty(string propertyName, string propertyValue)
+        /// <summary>
+        /// List based on https://github.com/nunit/docs/wiki/Attributes
+        /// </summary>
+        private readonly List<string> internalProperties = new List<string>
+            { "Timeout", "Repeat", "Apartment", "Author", "Combinatorial", "Culture", "Datapoint",
+              "DatapointSource", "DefaultFloatingPointTolerance", "Description", "LevelOfParallelism",
+              "MaxTime", "NonParallelizable", "Order", "Pairwise", "Platform", "Random", "Range",
+              "RequiresThread", "Retry", "Sequential", "SetCulture", "SetUICulture", "SingleThreaded", "Values", "ValueSource" };
```

Comment:
These should map to the properties in the NUnit class `PropertyNames`. It would be a bit easier if they were in alphabetical order, but I think the list should be 

```c#
"Author", "ApartmentState", "Description", "LevelOfParallelism", "MaxTime", "ParallelScope", "Repeat", "RequiresThread", "SetCulture", "SetUICulture", "TestOf", "Timeout", "IgnoreUntilDate", "Order"
```

The link should also point to https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Internal/PropertyNames.cs

Reply:
The difference is that not all attributes set properties, only those that derive from `PropertyAttribute` and those that do don't necessarily use the same names as the attributes themselves.

Reply:
Thanks, and Fixed :-)